internal/cmds/cmds.go

```diff
@@ -323,6 +331,80 @@ func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string) (shouldExit b
 	return false, nil
 }
 
+// Copy *.mrseq (Most Recently executed Sequence number) files from old extension version to new extension version during update.
```

Comment:
I recall we had a similar issue in RCv1, are we taking the same approach with RCv2?